src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.service;
+
+import mate.academy.exception.AuthenticationException;
+import mate.academy.model.User;
+
+public interface AuthenticationService {
+    User login(String email, String password) throws AuthenticationException;
+
+    User register(String email, String password) throws AuthenticationException;
```

Comment:
Do you think that changing interface because of implementation is a good idea? What about dependency inversion?

public interface AuthenticationService {
    User login(String email, String password) throws AuthenticationException;
    User register(String email, String password);
}

Reply:
конкретно в этом случае интерфейс был сыроват, логин тоже относится к аутентификации и тоже должен кидать эксепшн